asterisk/manager.py

```diff
@@ -458,7 +458,7 @@ def event_dispatch(self):
                 if callback(ev, self):
                     break
 
-    def connect(self, host, port=5038, buffer_size=0):
+    def connect(self, host, port=5038, buffer_size=0, timeout=60):
```

Comment:
According to the docs ... `settimeout()` method will accept `None` or a nonnegative floating point number, what do you think if we set

```suggestion
    def connect(self, host, port=5038, buffer_size=0, timeout=60.0):
```

For the default?